src/components/Calculator.js

```diff
@@ -1,14 +1,32 @@
 /*eslint-disable */
 import React, { Component } from 'react';
-
+import calculate from '../logic/calculator.js';
```

Comment:
- [ ] It would be great if this same script import calculates, and `operate.js` too, I see you import inside calculate, but according to requirements, the `operate.js` should be imported in this React component, please, work on it. 🐶